__tests__/app.test.js

```diff
@@ -56,6 +56,14 @@ describe("GET /api/articles/:article_id", () => {
         });
       });
   });
+  test("Respond with an article containig the comment_count property", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({ comment_count: 2 });
```

Comment:
I think it would be useful to add the article_id here to show the article received here is the correct one